tests/system/common.py

```diff
@@ -763,3 +763,11 @@ def task_by_name(tasks, name):
             return task
 
     assert False, "Did not find task with name %s in this list of tasks: %s" % (name, tasks,)
+
+
+@retrying.retry(wait_fixed=100, stop_max_attempt_number=30, retry_on_exception=ignore_exception, stop_max_delay=30000)
+def assert_event(event_type, events):
```

Comment:
assert_event_type maybe? Because we are not asserting any contents of the event...

Reply:
`assert_event_type` sounds good